tests/complex.test.js

```diff
@@ -385,7 +447,7 @@ var tests = [{
     set: "0-0i",
     fn: "pow",
     param: 0,
-    expect: "0"
+    expect: "1"
```

Comment:
What is the rationale behind "0^0"=1?

Reply:
Yep, that makes sense. But typically 0^0 is undefined. So `0` wasn't the best option either, but the question is if 0^0 should be defined as 1. I mean, `Math.pow(0,0)=1` having it for complex numbers equal to one would make it consistent.

Reply:
It's a very good point.  It seems like it should be either `Complex.ONE` or `Complex.NAN`.  The IEEE floating point standard defines it both ways according to wikipedia: https://en.wikipedia.org/wiki/Zero_to_the_power_of_zero#IEEE_floating-point_standard

Reply:
I would say let's go with `Complex.ONE` with a small note in the readme.